packages/gen-ai-hub/src/orchestration/orchestration-client.ts

```diff
@@ -47,6 +50,10 @@ export function constructCompletionPostRequest(
         },
         llm_module_config: input.llmConfig
       },
+      ...(input.filterConfig &&
+        input.filterConfig?.input &&
+        input.filterConfig?.output &&
```

Comment:
I think this is an issue, because only setting one of the two, or even setting none [is allowed](https://github.tools.sap/AI/llm-orchestration/blob/main/src/spec/api.yaml#L288). But the current code leave the filter config empty, if one of the two is not defined, right?